### DISCUSSION TITLE
Provide Default (Static) Tables

### DISCUSSION THREAD

**Discussion by funkyferdy** [Budibase/budibase#7765] [Ideas]
From my Experience, when you create some sort of Applications you come to a point where it would be handy to have a sort of 
"Default Static Tables" that come "handy" to have and "load quick" into your project (in Budibase Context = App). 

Think about countries, Regions, Languages, Currencies, etc. you name it. Sometimes you need this data to populate a dropdown in a form, as example.

Would be great if Budibase would ship allready such tables in a way that i can automaticly import in my app. As next step maybe even cooler if i can have one "instance wide" set that, later on, i can syncronise with instances (copys of this tables) used in app. Think about when a country, or a currency "disapears" and you have 35 Apps using the currency or Country Table.

In another project (TYPO3) there was this wonderful extension https://docs.typo3.org/typo3cms/extensions/static_info_tables/stable/Manual/Index.html#tables-structure that worked exactly this way.

Could be another USP and productivity booster for budibase "developers".

**Comment by melohagan** [comment]
Hey @funkyferdy 

This has been discussed internally, and I think the intention is to do something along these lines for brand new apps to have the option to include some 'dummy' data if the user wishes.

**Comment by eftomi** [comment]
Alternatively, this could be solved by services like https://restcountries.com/, https://countrylayer.com/ ...
